### ISSUE TITLE
M> FAQ

### ISSUE THREAD

**Issue by patrick727** [rchain/Members#60]
M> Indicates that this task is in the "Marketing" Category 

Goal:

There was a FAQ started and mention for one as we onboard more members. If someone wants to get take the lead with that would be awesome.

**Comment by kitblake** [comment]
I'll pick that up. I'm focusing on onboarding as it's a crucial issue. Where's the doc?

**Comment by patrick727** [comment]
thanks kit i'll add a bounty into the content sheet for this as well. here is the first draft back from redemption, some of these questions may not be applicable any more, and may  need fresh start. 

https://docs.google.com/document/d/1Q6AJRiMlMt-dVGQEEYQjNMMmhT-k8Lx1HhZQN9-s4vw/edit

**Comment by kitblake** [comment]
I have a draft FAQ ready. It needs polishing but review and editorial changes can happen in parallel. 
For instance, somebody who experienced the history needs to look at the "How did RChain come to be?" question, as I've skirted the contentious aspects.
Have a look, correct current content, think of new questions..
https://docs.google.com/document/d/1Q6AJRiMlMt-dVGQEEYQjNMMmhT-k8Lx1HhZQN9-s4vw/edit

**Comment by kitblake** [comment]
We have progress on the FAQ. @edeykholt jumped in :)

Next steps:
- If any other BOD/Activists want to chime in please do it soon.
- Otherwise, when we think we have a suitable version I'd like to post the edit link in Slack, like in the Coop channel, to gather reactions from the community at large.
- I presume there are questions we haven't thought of that would be good to have in the FAQ. Certainly the developers section could use some input (**nudge, nudge**). Plus trolls may have nasty questions that we can either address or ignore.
- After community feedback and some polishing we'll have a final FAQ and we'll need to decide where it should be published. If it's on the website then I'd like to see an interactive UI (@patrick727 already knows this ;)

**Comment by patrick727** [comment]
@kitblake greatjob thanks. I think that it should be posted as a wiki in github, or a repo in which we can link to in the reference board. If people wish to add questions then we can just add an issue or pull request so that there is some process. 

What are your guys thoughts?

**Comment by kitblake** [comment]
Done, it's posted. 
@patrick727, I notice that the FAQ content has gotten ahead of the website (e.g. BoD officers) so we should make sure to sync the content when the FAQ goes public.

**Comment by patrick727** [comment]
Yes both the website and the FAQ(any messaging) should be consistent and up to date to the best of our abilities, what exactly do you mean by BOD officers as your example?

**Comment by kitblake** [comment]
In https://www.rchain.coop/coop-information-1#board-of-directors it says: 
"2 Board of Director spots remain, and Officers are still to be chosen." 
while the FAQ says:
"The Co-op currently has a five-person Board of Directors. The officers are Greg Meredith, President, and Evan Jensen, Secretary. ... Additional board members and officers **may** be added."

**Comment by patrick727** [comment]
@kitblake good catch, I edited the site to reflect the most recent information from the FAQ

**Comment by kitblake** [comment]
If – and that's def an if – the FAQ goes on the website, here's an example of what I mean by interactive UI, where it all happens onclick:
https://pillarproject.io/faq/

However, as @patrick727  suggested, placing the FAQ in a repo on Github would be fine. In any case the content of the FAQ should be moved from the GDoc to Github, converted to Markdown, and maintained there. 

Remember that Markdown renders as HTML in Github's web UI. The FAQ won't be interactive and it'll be a long page but it is what it is. Making a Github repo the published location of the RChain FAQ has a certain resolute charm.

**Comment by patrick727** [comment]
@kitblake I like the interactive UI but am also fine with the longer page like post that constantly is iterating. 
It seems that in the debrief that a Github post was a reasonable way to go. and yes charming indeed and keeps with where the activists are participating. :) Great job

**Comment by lapin7** [comment]
Great advance!!

HJ

**Comment by kitblake** [comment]
Yes, the Debrief reaction was fine, kudos to all.
@plantether, thanks for your comments. Some of them require BoD-level review.
Btw, the Developers section is still kinda scant....
I'll post a 'last call' in Slack for feedback, then we can migrate to Github.

**Comment by kitblake** [comment]
Migrating the FAQ content to GitHub..

**Comment by kitblake** [comment]
FAQ published: https://github.com/rchain/reference/blob/master/faq.md
Closing this issue.

**Comment by lapin7** [comment]
Currently the budget for this issue is:  ⍴ 8,333.00

I think @kitblake suggested to pay like this:
But @plantether and @kdvalentine didn't fill out their column in [Gsheet "RChain Activist RHOC Distribution \ RHOC rewards August 2017"](https://docs.google.com/spreadsheets/d/1wD8DzlvXZCtfycToN0K506Kev4oJm8yvosY0FIHZiYM/edit#gid=846811289).
And maybe Activists have to be added to this issue, by copying and pasting a row under the three existing rows for issue #60

⍴ To Pay | Activist | AVG work
-- |  -- | --
⍴ 6,667 | @kitblake | 80%
⍴ 417 | @planteater | 5%
⍴ 167 | @kdvalentine | 2%

It's important to get these decisions and payments going ;-)

**Comment by kitblake** [comment]
For clarity's sake (because I didn't get it at first) the columns per individual in the "RHOC rewards August 2017" sheet work the same as in the "Budget Allocation" sheet: i.e. the percentages that we fill in get averaged.

So both the Budget Allocations *and* the RHOC Rewards are an average of what your peers say.

This works nicely. We all guesstimate a budget, then when the project concludes the team members estimate their participation, and that determines the payout.

**Comment by kitblake** [comment]
The faq.md doc has 5 pull requests in the queue. Can somebody deal with them before the next major edit?  
https://github.com/rchain/reference/blob/master/faq.md

**Comment by Ojimadu** [comment]
Would attempt to fix it. Let me take a look.

**Comment by kitblake** [comment]
Thanks. There's an edit from @drbloom about Discord that should supersede the earlier one.

**Comment by kitblake** [comment]
Thanks for dealing with those pull requests. 
A lot has changed and the FAQ needs work so I'm reopening this issue.

**Comment by kitblake** [comment]
Just checked in an intermediary update:
https://github.com/rchain/reference/pull/15

**Comment by lapin7** [comment]
New Questions that evaluate RChain:
How good is RChain?

**Website: **

- Does it look good/presentable?
- Are the links informative?
- Does the website have a roadmap for the cryptocurrency development?
- Is there a Trello board https://trello.com/
- Does it have a whitepaper.
- Is the whitepaper good? explaining problems and own solutions
- Is the domain name good?
- Is it a quick wordpress template, a 2-hour html job or a custom design with lots of work, money and thought thrown at it?
- What new does this cryptocurrency bring to the world, what problem does it solve, what does it stand for?
- How does the cryptocurrency name sound (does it sound legit and like a good brand name)
- How does the logo look?.
- Is it a PoW, PoS or PoW/PoS coin, what is the mining difficulty or APR, does it suffer from inflation – pre-minted. 
- Who is behind the cryptocurrency – are they shady figures, or have they placed their names and pictures on the website; 
- What is their track record and reputation?
- The amounts of money they’ve raised so far. 
- Who's the founder?
- Teammembers?
- What is the limit on supply of coins in circulation?
- Is there a link to the source code?
- Github activitity?
- How many developers participate in the coding process
- Regulatory permissions?.

**Exchanges and trading**

- Which exchanges (and how reputable they are) is the coin traded on, what is the daily trading volume and market cap?
- Are there buy (support) or sell (resistance) walls?
- How does the chart look?
- Does the chart contain patterns that look like pump and dump actions or trading by bots?
- Has the coin spiked on one exchange after being added on another exchange?

**Social**

- Is there a Bitcointalk forum Announcement thread 
- How many pages is it? Summary
- Is there a blog, anything interesting on it?
- Is there a Discord (Slack) channel
- How many members in the slack channel?
- What is the public sentiment in the slack channel?
- Is there a Twitter profile for the cryptocurrency and does it post regular updates, how many followers? Daily tweets
- Is there a Facebook page for the cryptocurrency and does it post regular updates, how many likes Twitter activity

**Current state**

- Are there any known problems with the source code, the wallet or something else regarding the coin (transactions, mining, efficiency)?
- Is there an active development for fixes on the way?
- Has the wallet been under maintenance for prolonged periods of time on some/all of the - exchanges?
- Is any government planning on banning the coin (or similar coins) or putting heavy restrictions towards its use?

**Popularity and perception**

- What does a quick google search tell us about the coin?
- Are there youtube videos about the coin?
- What is the general sentiment in the comment section below the videos?
- Is the coin associated with illegal/criminal activity or the dark web?
- Similar cryptocurrencies/projects?

**Comment by patrick727** [comment]
@kitblake on call yesterday Ed went through and accepted pull requests I believe. 

Thanks for reopening and @lapin7 these are all great questions. 

We also have the need for a fully fleshed out Wiki to be published to the updated rchain.coop. 

Great stuff all!

**Comment by Ojimadu** [comment]
I was about creating an issue to aggregate outdated information on the website and constantly update the FAQ so all FAQs and information on the website would be the same and updates. These information can be aggregated and updated weekly or bi-weekly.
@lapin7  Nice great questions there. I would attempt writing answers for them and merging them with the FAQ am also compiling.

**Comment by kitblake** [comment]
Two fyi points:
1. This weekend I renovated the FAQ and sent a pull request this morning, but I neglected to reference this issue #60. 
2. I added the following at the end of the FAQ, which dovetails with the aggregation idea:

### *Suggestions?*
Have a question that hasn't been answered? You can suggest additional FAQ entries by including the hashtag #faq in any of the comm channels.

**Comment by Ojimadu** [comment]
@kitblake, Thanks would put those into consideration. 
@lapin7, are all those questions meant for the FAQ? Or are they meant for evaluation?  Because some of the questions are more evaluatory than explanatory.

**Comment by lapin7** [comment]
Those questions are indeed a kind of evaluation of RChain. I think they are
of interest to people who want to know a bit more about RChain.
Not all questions have to be answered. Skip as you see it fit.

On the other hand, those questions can be turned into issues.

--
Cheers,
HJ

**Comment by kitblake** [comment]
Hello Ian, Ed, Greg, Joe, Navneet, Mike, 
aka @drbloom, @edeykholt, @leithaus, ?, @n10n, @pyrocto, 

We have pull requests for the FAQ queuing up, and I fear some will trigger conflicts. 
Nothing complicated for a coder but what's needed is content review. 
Can one of you move it forward? 

Once this version gets published a translation to Chinese is in the works.

**Comment by pyrocto** [comment]
There are no pull requests that I can see to review.  Do you mean some project other than rchain/Members?

**Comment by kitblake** [comment]
Sorry, should've linked it. The pull requests are in /reference:
https://github.com/rchain/reference/pulls

**Comment by kitblake** [comment]
Hello Ian, Ed, Greg, Joe, Navneet,
aka @drbloom, @edeykholt, @leithaus, ?, @n10n,

@pyrocto responded with some welcome developer feedback that's now been incorporated. What's needed is content review before merge. 
Can somebody pick this up?

**Comment by kitblake** [comment]
The pull requests are in /reference for the FAQ:
https://github.com/rchain/reference/pulls

**Comment by Ojimadu** [comment]
M> Marketing
O> Operations
F> Finance
L> Legal
D> Development

Was thinking of where to add the issues markers above. Whether it should be added to the FAQs or the [contributing.MD](https://github.com/rchain/Members/blob/master/CONTRIBUTING.md) or both.

**Comment by n10n** [comment]
@kitblake don't see any pull request. Let me know when you have one. Here is another issue that I added. https://github.com/rchain/Members/issues/214.

**Comment by kitblake** [comment]
@n10n that nudge about a pull request on november 28th got merged on december 5th, so we're good. In the meantime there have been more, and they've been merged more quickly (so thanks). 

@Ojimadu imho the MOFLD tagging is for internal use, and not for a public facing document like the FAQ. Contributing.md is sorta in between, but more on the public side as it's for newbies.
